### ISSUE TITLE
Stack widget with 'overflow' property with latest flutter version (1.22)

### ISSUE THREAD

**Issue by Bhargav3132** [darioielardi/flutter_speed_dial#84]
pub.dartlang.org/flutter_speed_dial-1.2.5/lib/src/speed_dial.dart:249:7: Error: No named parameter with the name 'overflow'.
      overflow: Overflow.visible,
      ^^^^^^^^
../../../Flutter/flutter/packages/flutter/lib/src/widgets/basic.dart:3273:3: Context: Found this candidate, but the arguments don't match.
  Stack({

**Comment by krumpli84** [comment]
> pub.dartlang.org/flutter_speed_dial-1.2.5/lib/src/speed_dial.dart:249:7: Error: No named parameter with the name 'overflow'.
> overflow: Overflow.visible,
> ^^^^^^^^
> ../../../Flutter/flutter/packages/flutter/lib/src/widgets/basic.dart:3273:3: Context: Found this candidate, but the arguments don't match.
> Stack({

Can't archive on iOS due to this...

**Comment by nguyendinhthuc18** [comment]
please fix it

**Comment by LachlanGrant** [comment]
Looks like it's already been fixed here: https://github.com/darioielardi/flutter_speed_dial/pull/83

```
  flutter_speed_dial:
    git:
      url: https://github.com/irasekh3/flutter_speed_dial
      ref: master
```

Full credit to @irasekh3

**Comment by prateekmedia** [comment]
Checkout V2 #90 it will be merged soon, it also has #83 implementation credits goes to @irasekh3 for this.
